ops/createComponent.test.js

```diff
@@ -29,17 +29,32 @@ describe(`createComponent`, () => {
 
     it(`should throw an error`, async () => {
       try {
-        await createComponent({ path: `one`, name: `myComponent` })
+        await createComponent({ path: `app/one`, name: `myComponent` })
         throw new Error(`It did not throw an error`)
       } catch (e) {
         expect(e.message).to.equal(`A component 'myComponent' already exists`)
       }
     })
   })
 
+  describe(`when create outside app dir`, () => {
+    beforeEach(async () => {
+      await mkdirp(`app/one/two/three`)
+    })
+
+    it(`should throw an error`, async () => {
```

Comment:
When in root folder it should assume `app`.

Just as a convenience thing. Because most commands will be executed in root folder.